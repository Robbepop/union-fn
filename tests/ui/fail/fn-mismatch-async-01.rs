fn main() {}

#[union_fn::union_fn]
trait Foo {
    fn foo() {}
    async fn bar() {}
}
