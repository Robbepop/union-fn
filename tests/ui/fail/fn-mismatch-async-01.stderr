error: encountered mismatch in asyncness for #[union_fn] method
 --> tests/ui/fail/fn-mismatch-async-01.rs:6:5
  |
6 |     async fn bar() {}
  |     ^^^^^

error: mismatch with this method
 --> tests/ui/fail/fn-mismatch-async-01.rs:5:5
  |
5 |     fn foo() {}
  |     ^^
